clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/DefaultOrchestrationProcessor.groovy

```diff
@@ -201,4 +212,42 @@ class DefaultOrchestrationProcessor implements OrchestrationProcessor {
     summary.putAll(additionalFields)
     return summary
   }
+
+  @Nonnull
+  private GetTaskResult getTask(String clientRequestId) {
+    def existingTask = taskRepository.getByClientRequestId(clientRequestId)
+    if (existingTask) {
+      if (!isTaskRetryable(existingTask)) {
+        return new GetTaskResult(existingTask, false)
+      }
+      existingTask.updateStatus(TASK_PHASE, "Re-initializing Orchestration Task...")
```

Comment:
Minorest of nits, but I think this would be better without the `...`

Reply:
Agreed. I've been removing those from other places but missed these. Thanks!